hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystemStore.java

```diff
@@ -253,6 +259,20 @@ public boolean getIsNamespaceEnabled() throws AzureBlobFileSystemException {
     return isNamespaceEnabled;
   }
 
+  @VisibleForTesting
+  boolean isNameSpaceEnabledSetFromConfig() {
```

Comment:
This method should be in AbfsConfiguration.java ?

Reply:
No.
Though the possible values for the config is true/false, we cannot have a default value for the same. So we accept the config value as String and if it is valid(true/false) we set it to the AzureBlobFileSystemStore.isNamespaceEnabled.

If the config is not present or the value specified is invalid, this method returns false, indicating it could not set the AzureBlobFileSystemStore.isNamespaceEnabled field.

And according to the return value of this method, the caller method returns AzureBlobFileSystemStore.isNamespaceEnabled field or falls back to the default behaviour.

Reply:
Done